sw/host/opentitantool/src/command/update_usr_access.rs

```diff
@@ -36,10 +36,11 @@ impl CommandDispatch for UpdateUsrAccess {
         _transport: &TransportWrapper,
     ) -> Result<Option<Box<dyn Annotate>>> {
         let mut bs = fs::read(&self.input)?;
-        usr_access_set(
-            &mut bs,
-            self.usr_access.unwrap_or_else(usr_access_timestamp),
-        )?;
+        let mut usr_access_val = self.usr_access;
+        if self.usr_access.is_none() {
```

Comment:
PLMK if there's a better way to do this. I couldn't find a good counterpart to `unwrap_or_else` that allows the called function to return an error.

Reply:
I think you can use the question mark operator on the right-hand side of a case in a match expression. The side benefit is that `usr_access_value` doesn't need to be mut anymore :)